src/Altinn.ApiClients.Maskinporten/Altinn.ApiClients.Maskinporten.csproj

```diff
@@ -2,9 +2,15 @@
 
   <PropertyGroup>
     <TargetFramework>net5.0</TargetFramework>
-    <Version>0.9.62-alpha</Version>
+    <Version>1.0.0</Version>
```

Comment:
Is this being used now? Isn't the version picked up from the tag?

Reply:
No, this is not used as the `dotnet pack` overrides it so I suppose this can be removed.